main.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_media_convert_queue" "queue" {
+  name         = var.queue_name
+  pricing_plan = var.queue_pricing_plan
+
+  status = var.status
+
+}
+
+module "iam_role" {
+  source  = "dasmeta/iam/aws//modules/role"
+  version = "1.2.1"
+
+  name = "mediaconverter-${var.iam_role_name_suffix}"
```

Comment:
can we use var.queue_name as identifier here instead of var.iam_role_name_suffix so that we can remove var.iam_role_name_suffix ?